src/App.js

```diff
@@ -22,6 +23,7 @@ function App() {
         <Route path="/about-us" element={<AboutUs />} />
         <Route path="/my-cart" element={<MyCart />} />
         <Route path="/login" element={<Login />} />
+        <Route path="/product-detail" element={<ProductDetailPage />} />
```

Comment:
For dynamic URL by product id use this

```javascript
<Route
          path="/product-detail/:productId"
          element={<ProductDetailPage />}
        />
```

And use `useParams` hook for getting `productId` on the product detail page.